indico/modules/events/registration/controllers/management/eticket.py

```diff
@@ -0,0 +1,39 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import redirect
+
+from indico.core.db import db
+from indico.modules.events.registration.controllers.management import RHManageRegFormBase
+from indico.modules.events.registration.forms import RegistrationFormETicketForm
+from indico.modules.events.registration.views import WPManageRegistration
+from indico.web.flask.util import url_for
+from indico.web.forms.base import FormDefaults
+
+
+class RHRegistrationFormETicket(RHManageRegFormBase):
+    """Display"""
+
+    def _process(self):
+        form = RegistrationFormETicketForm(obj=FormDefaults(eticket_enabled=self.regform.eticket_enabled))
```

Comment:
`ETicketForm` is more than enough, and you can simply pass `obj=self.regform` instead to get the default